draft-ietf-scitt-architecture.md

```diff
@@ -624,12 +623,12 @@ Append-Only:
 
 Non-equivocation:
 
-: there is no fork in the Append-only Log.
+: there is no fork in the Verifiable Data Structure.
```

Comment:
```suggestion
: there is no fork in the Append-only Log.
```

The VDS is there to make forks obvious, but the thing we are protecting is the log itself, not the tree.

Reply:
A fork does not even make sense in the context of non linear VDS, so I agree that some change is needed here.

I think a better solution would be to frame "Non-equivocation" in terms of the VDS that does not assume a linear log... for example:

A property of certain implementations of VDS preventing conflicting statements between different processes.
In Byzantine fault tolerant systems, this property can be achieved with trusted hardware.

Reply:
If the existing PR or Suggestion isn't good enough, please make an alternative suggestion.

Reply:
I have tried once more, with "sequence of Signed Statements", which is the log in my mind but is hopefully a more widely acceptable term. Does that work?

Reply:
Note: Without weighing in on the suggestion, I've updated the suggestion from TS --> Transparency Service.